### DISCUSSION TITLE
新たなサイレンス機能の提案

### DISCUSSION THREAD

**Discussion by tkmrgit** [misskey-dev/misskey#10010] [Ideas]
関連 #9862 かも
Issueではないと思ったので初めてこちらで。ioは大量流入によりLTLが物凄い流れになっており、それで内容によってはMisskeyに来て文化が分からない方には一部不快感を持っているユーザーもおられるのかなと思います。そこで、Misskeyの機能としてモデレーション機能の強化を提案します。どれが正解かは私も悩むので、以下はあくまでも提案という形で思いついたものを記します。ほかの方による提案も大歓迎です。

### ★新サイレンス
現在のサイレンスは行われるとLTLにまったく表示されなってしまい、投稿範囲がホーム以下しか行えなくなってしまう。**警告が表示されてもそのまま投稿してMFMを多用しているユーザーや複数行での投稿を繰り返し行っているユーザーを片っ端からサイレンスにするのは賢明ではない**可能性があるので、LTLには表示されない(がフォローしているユーザーがいる場合はLTLにも表示されても良いかもしれない。STLを使えばいいじゃないかと思うかもしれないがSTLを積極的に使うユーザーがどれだけ多いかは不明な為)、対象ユーザーがサイレンスされたのがわかりにくく(ロールで実装されるとわかるかも?)、またサイレンスされたことでアカウントを新たに作ることをある程度防止できる、公開範囲がホーム以下にならないような新たなサイレンス機能(ソフトサイレンス?)を提案します。

内容を3行で解説すると
- LTLには表示されなくなることで不快感が薄まる
- ただしその対象アカウントや対象アカウントをフォローしているユーザーの場合はLTLに表示されても良い?
- 公開範囲がホーム以下にならないサイレンス機能

**Comment by rinsuki** [comment]
面倒だしみんなSTL見てそうだしSTL見てねでよくない？

**Reply by tkmrgit** [reply]
ありがとうございます。STL見てねで従ってくれる人がどれだけいるかなという問題と、既存サイレンスの問題点(ホーム以下になるのでわかってしまう)を解決するにはどうしたら良いか？を考えた提案となっています。確かに対応は面倒かもしれません。

**Comment by acid-chicken** [comment]
#9325 がより大変になるし STL 使ってほしい
